plugins/usbdmx/VellemanK8062.cpp

```diff
@@ -401,7 +403,7 @@ void VellemanAsyncUsbSender::PostTransferHook() {
     } else {
       // No pending transfer. The widget only actually sends a frame once the
       // next frame begins, so kick off the next frame here.
-      PerformTransfer(m_tx_buffer);
+      PerformTransfer(m_tx_buffer, 0);
```

Comment:
This is pretty much a hack since that device has only one output port anyways ...

Reply:
Seems to me in your paradigm the PostTransferHook should simply get passed the portId too, no? I think that would imply having the tranciever remember the portId for a transfer which could get tricky since they don't currently deal with per-transfer data, but rather just pass their own `this` pointers to libusb as user data for the transfer.

See AsyncUsbTransceiverBase::FillControlTransfer for example. Notice the `this` argument.